scripts/aws.sh

```diff
@@ -0,0 +1,488 @@
+          echo "fs.inotify.max_user_instances = 8192" >> /etc/sysctl.conf
+          sysctl --system |& tee -a /tmp/init_output.txt
+          sysctl -a |& tee -a /tmp/init_output.txt
+
+Outputs:
+  InstanceId:
+    Description: RHEL Host Instance ID
+    Value: !Ref RHELInstance
+  PrivateIp:
+    Description: The bastion host Private DNS, will be used for cluster install pulling release image
+    Value: !GetAtt RHELInstance.PrivateIp
+  PublicIp:
+    Description: The bastion host Public IP, will be used for registering minIO server DNS
+    Value: !GetAtt RHELInstance.PublicIp
+EOF
+
+  aws --region "${region}" cloudformation create-stack --stack-name "${stack_name}" \
+    --template-body "file://${cf_tpl_file}" \
+    --capabilities CAPABILITY_NAMED_IAM \
+    --parameters \
+    ParameterKey=HostInstanceType,ParameterValue="${inst_type}" \
+    ParameterKey=Machinename,ParameterValue="${stack_name}" \
+    ParameterKey=AmiId,ParameterValue="${ami_id}" \
+    ParameterKey=EC2Type,ParameterValue="${ec2Type}" \
+    ParameterKey=PublicKeyString,ParameterValue="$(cat "${pub_key}")"
+    
+  #echo "Waiting for stack to be created"
+  aws --region "${region}" cloudformation wait stack-create-complete --stack-name "${stack_name}" 2> "${err_log}" &
+  pid=$!
+  spin "${pid}" "Waiting for stack to be created" "Stack created successfully"
+
+  # shellcheck disable=SC2016
+  instance_id="$(aws --region "${region}" cloudformation describe-stacks --stack-name "${stack_name}" \
+    --query 'Stacks[].Outputs[?OutputKey == `InstanceId`].OutputValue' --output text)"
+
+  aws --region "${region}" ec2 wait instance-status-ok --instance-id "${instance_id}" 2> "${err_log}" &
+  pid=$!
+  spin "${pid}" "Waiting for stack status to be OK" "Stack status OK"
+
+  # shellcheck disable=SC2016
+  public_ip=$(aws --region "${region}" cloudformation describe-stacks --stack-name "${stack_name}" \
+    --query 'Stacks[].Outputs[?OutputKey == `PublicIp`].OutputValue' --output text)
+
+  rm -f "${cf_tpl_file}"
+  
+  echo "PUBLIC IP: ${public_ip}"
+}
+
+action_delete() {
+  aws --region "${region}" cloudformation delete-stack --stack-name "${stack_name}"
+	aws --region "${region}" cloudformation wait stack-delete-complete --stack-name "${stack_name}" 2> "${err_log}" &
+  pid=$!
+  spin "${pid}" "Waiting for stack to delete" "Stack deleted"
+	exit 0
+}
+
+action_describe() {
+  aws --region "${region}" cloudformation describe-stack-events --stack-name "${stack_name}" --output json | less
+	exit 0
+}
+
+action_logs() {
+  # shellcheck disable=SC2016
+  instance_id="$(aws --region "${region}" cloudformation describe-stacks --stack-name "${stack_name}" \
+    --query 'Stacks[].Outputs[?OutputKey == `InstanceId`].OutputValue' --output text)"
+  
+  aws ec2 get-console-output --instance-id "${instance_id}" --output text
+}
+
+usage() {
+  cat - <<EOF
+Script for AWS stack management
+
+Usage:
+
+aws.sh (create|delete|describe|logs) \ 
```

Comment:
Should we call the script `manage_aws_stack.sh` like other similar ones?
```suggestion
$(basename $0) (create|delete|describe|logs) \ 
```

Reply:
Done